atlas/AtlasMl/atlasml/app.py

```
@@ -97,3 +113,7 @@ async def validation_exception_handler(request: Request, exc: RequestValidationE
 
 app.include_router(health_router)
 app.include_router(competency_router)
+
+@app.get("/sentry-debug")
+async def trigger_error():
+    division_by_zero = 1 / 0
```

Ruff (0.12.2)

Local variable `division_by_zero` is assigned to but never used

Remove assignment to unused variable `division_by_zero`

(F841)